frontend/src/components/filter/FilterNationalCenterNameSelect.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { getNationalCenters } from '../../fetchers/nationalCenters';
+import FilterSelect from './FilterSelect';
+import { filterSelectProps } from './props';
+
+export default function FilterNationalCenterNameSelect({
+  onApply,
+  inputId,
+  query,
+  title,
+}) {
+  const [nationalCenters, setNationalCenters] = useState([]);
+
+  useEffect(() => {
+    async function fetchNationalCenterNames() {
+      const { centers } = await getNationalCenters(title);
```

Comment:
looks like you can remove title here since it isn't used (that will make the use effect dependencies simpler as well)

Reply:
please wrap the interior of this function in a try/catch block

Reply:
done.